docs/application/dotnet/index.md

```diff
@@ -42,7 +42,7 @@ Tizen .NET consists of the following main components:
 
 - **.NET Core**
 
-  [.NET Core](https://docs.microsoft.com/en-us/dotnet/core/about) is an [open-source](https://github.com/dotnet/coreclr/blob/master/LICENSE.TXT), general-purpose development platform maintained by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It's cross-platform (supporting Windows, macOS, and Linux) and can be used to build device, cloud, and IoT applications.
+  [.NET Core](https://docs.microsoft.com/en-us/dotnet/core/about) is an [open-source](https://github.com/dotnet/coreclr/blob/master/LICENSE.TXT), general-purpose development platform maintained by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It is cross-platform (supporting Windows, macOS, and Linux) and can be used to build device, cloud, and IoT applications.
```

Comment:
Suggestion:
Could we also add information regarding OpenTK here. If yes, please provide more information on the same.

Reply:
We carefully discussed about it, but decided not to include information regarding OpenTK.

Reply:
@hakkyu-samsung,

Thank you for the clarification.